### PR TITLE
Measure $evaluate-measure operation

### PR DESCRIPTION
**Pull request by codyebberson**
[`$evaluate-measure`](https://hl7.org/fhir/measure-operation-evaluate-measure.html) FHIR operation

Link to Medplum docs preview: https://medplum-www-git-cody-evaluate-measure-medplum.vercel.app/docs/api/fhir/operations/evaluate-measure

In service of https://github.com/medplum/medplum/issues/2009

### CODE REVIEW

File: packages/docs/docs/api/fhir/operations/evaluate-measure.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+      ]
+    }
+  ]
+}
+```
+
+Next, make an HTTP POST request to the `$evaluate-measure` endpoint:
+
+```bash
+curl 'https://api.medplum.com/fhir/R4/Measure/MY_MEASURE_ID/$validate' \
```
Comment (tonylee80) [inline packages/docs/docs/api/fhir/operations/evaluate-measure.md:43]: typo `validate` -> `$evaluate-measure`

**Review by tonylee80 — APPROVED** [review]

**Comment by tonylee80** [comment]
Looks good to me!

File: packages/docs/docs/api/fhir/operations/evaluate-measure.md (hunk 1 above)
Comment (codyebberson) [reply packages/docs/docs/api/fhir/operations/evaluate-measure.md:43]: eek!  ty!

**Comment by mattwiller** [comment]
This looks like it's correctly calculating the population counts, but I didn't see anywhere where the `Measure.library` was actually evaluated to score the patients in the population.  We may want to call that out as a caveat until we can add full support for the endpoint functionality, since that's usually how these measures tend to get used I believe

File: packages/docs/docs/api/fhir/operations/evaluate-measure.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+`$evaluate-measure` is early and limited. At present, it only supports calculating counts using "application/x-fhir-query". Using FHIR `Library` is currently unsupported.
+
+:::
+
+## Usage
+
+First, create a FHIR `Measure`:
+
+```json
+{
```
Comment (rahul1) [inline packages/docs/docs/api/fhir/operations/evaluate-measure.md:20]: Can we move the examples to the `packages/examples` directory

Comment (codyebberson) [reply packages/docs/docs/api/fhir/operations/evaluate-measure.md:20]: Hm, the advantage of the `examples` directory is type checking, and these are just JSON?

**Pull request merged**